src/tezos/operation.ml

```diff
@@ -3,16 +3,11 @@ open Data_encoding
 
 let () = Printexc.record_backtrace true
 
-type parameters = {
-  entrypoint : string;
-  value :
-    Michelson_v1_primitives.prim Tezos_micheline.Micheline.canonical
-    Data_encoding.lazy_t;
-}
 type transaction = {
   amount : Tez.t;
   destination : Address.t;
-  parameters : parameters option;
+  entrypoint : string;
```

Comment:
At this point, I am not sure about what is _really_ the entrypoint (I think it is _the smart contract entrypoint_, am I right?), but should we go for a custom type instead of `string`?
I think going for a custom type could help us to get some domain specific: What is this string, functionally? => Can this string be empty? Can this string be only a space character? Does it have to match a given regex?

Reply:
Yes, this is the contract entrypoint. About the functionality, it can be any string if I remember correctly up to 61 chars.

Your argument about enforcing it on the typer makes sense, just that it's not a considerable win here, but an considerable amount of complexity, especially because this entrypoint is likely gonna be a hard coded string.

I would say that this should not block this PR, but if you want to open another PR adding an `Entrypoint.t` feel free.

Reply:
Opened #452